New Bank/NewBank/newbank/server/MicroLoan.java

```diff
@@ -0,0 +1,35 @@
+package newbank.server;
+
+public class MicroLoan {
+
+    private double amount;
+    private String repaymentTerm;
+
+    public double getAmount() {
+        return amount;
+    }
+
+    public void setAmount(double amount) {
+        this.amount = amount;
+    }
+
+    public String getRepaymentTerm() {
+        return repaymentTerm;
+    }
+
+    public void setRepaymentTerm(String repaymentTerm) {
+        this.repaymentTerm = repaymentTerm;
+    }
+
+    public double getCurrentIncome() {
+        return currentIncome;
+    }
+
+    public void setCurrentIncome(double currentIncome) {
+        this.currentIncome = currentIncome;
+    }
+
```

Comment:
The 'getters' in this class are not used. It's probably fine to keep them in, as they might be useful later, but I thought I'd mention it just in case this wasn't intentional.

Reply:
noted.